src/App.css

```diff
@@ -1 +1,49 @@
-/* not empty */
+[data-cy='item'] {
+  list-style: disc;
+}
+
+.active {
+  color: white;
+  background-color: blue;
```

Comment:
Direct color values like 'white' and 'blue' can lead to inconsistencies across the application. Consider using CSS variables or a predefined set of color constants to ensure consistency and make future changes easier.